src/core/qfieldcloudprojectsmodel.cpp

```diff
@@ -1081,7 +1081,15 @@ void QFieldCloudProjectsModel::projectDownload( const QString &projectId )
       }
       else
       {
-        if ( !layer.value( QStringLiteral( "is_valid" ) ).toBool() && !layer.value( QStringLiteral( "is_localized" ) ).toBool() )
+        if ( layer.value( QStringLiteral( "is_localized" ) ).toBool() )
+        {
+          const QString localizedDataset = layer.value( "datasource" ).toString().mid( 10 );
```

Comment:
I think it would be even beter if the cloud returns you the proper localized filename within the project, so there is only one place that calculates this.

Reply:
@suricactus , maybe -- see https://github.com/opengisch/QFieldCloud/pull/1207